src/sentry/static/sentry/app/components/forms/inputField.jsx

```diff
@@ -43,7 +43,7 @@ export default class InputField extends FormField {
   }
 
   getId() {
-    return 'wizard-' + this.props.name;
+    return 'id-' + this.props.name;
```

Comment:
I erronously assumed these fields were only used by the install wizard, so naming this something generic.